.github/workflows/build-and-run-model.yaml

```diff
@@ -69,8 +69,9 @@ jobs:
       # required in order to allow the reusable called workflow to push to
       # GitHub Container Registry
       packages: write
-    uses: ccao-data/actions/.github/workflows/build-and-run-batch-job.yaml@main
+    uses: ccao-data/actions/.github/workflows/build-and-run-batch-job.yaml@jeancochrane/20-container-env-var-masks-can-cause-cloudwatch-logs-link-to-get-masked
```

Comment:
Once https://github.com/ccao-data/actions/pull/21 is merged, I'll switch this back to `main` before merging this PR.